Userland/Libraries/LibJS/Bytecode/Generator.cpp

```diff
@@ -9,8 +9,8 @@
 #include <LibJS/Bytecode/Block.h>
 #include <LibJS/Bytecode/Generator.h>
 #include <LibJS/Bytecode/Instruction.h>
-#include <LibJS/Bytecode/Register.h>
 #include <LibJS/Bytecode/Op.h>
+#include <LibJS/Bytecode/Register.h>
```

Comment:
This include is added in the first commit and then ordered properly in the fourth, please avoid changes like these across commits.